manager/container.go

```diff
@@ -335,7 +335,10 @@ func (cd *containerData) GetProcessList(cadvisorContainer string, inHostNamespac
 		vs *= 1024
 		psr, err := strconv.Atoi(fields[11])
 		if err != nil {
-			return nil, fmt.Errorf("invalid pid %q: %v", fields[1], err)
+			// to avoid defunct processes
+			//return nil, fmt.Errorf("invalid psr %q: %v", fields[11], err)
```

Comment:
I think we still want to fail on an error here, right?

Reply:
Thank you for the comment. The reason of the commit is "to increase fault tolerance of  cadvisor". Unfortunately, process list is returned as empty in such case, when any external process (even not in container) appeared in the host system as defunct. It is regular case. I think, cadvisor should take into account and handle it correctly. But, "psr = -1" was the easiest and maybe not the best solution. Do you have an alternative variant?

Reply:
What error message were you getting here?

Is this related to adding the additional field, or is this fixing a preexisting problem?

Reply:
This is fixing a preexisting problem. Error message `invalid psr %q: %v <defunct> strconv.Atoi: parsing "<defunct>": invalid syntax` appears on attempt to convert string "<defunc>" to integer via strconv.Atoi.

Reply:
can we check for "defunct" specifically instead of ignoring the error entirely?